android/libraries/rib-router-navigator/src/main/kotlin/com/uber/rib/core/RouterNavigator.kt

```diff
@@ -275,7 +260,8 @@ interface RouterNavigator<StateT : RouterNavigatorState> {
     }
 
     /**
-     * Notifies the consumer that the [StackRouterNavigator] has detached the supplied [ ]. Consumers can complete any post detachment behavior here.
+     * Notifies the consumer that the [StackRouterNavigator] has detached the supplied [ ].
+     * Consumers can complete any post detachment behavior here.
      *
      * @param router [Router]
```

Comment:
IMO we can remove all those @param that add nothing to the reader. They're ubiquitous in JavaDoc but not much common in KDoc.

Reply:
I will remove this and similar changes from this PR and will create separate one to do that.